lisa/tools/fdisk.py

```diff
@@ -44,10 +44,10 @@ def make_partition(self, disk_name: str, file_system: FileSystem) -> None:
         )
         if file_system == FileSystem.xfs:
             mkfs_xfs = self.node.tools[Mkfsxfs]
-            mkfs_xfs.mkfs(f"{disk_name}p1", str(file_system))
+            mkfs_xfs.mkfs(disk_name, f"{str(file_system)} -f")
```

Comment:
What the -f is used for?

Reply:
Write like below should be enough. The str() will be used automatically.
`f"{file_system} -f"`

Reply:
fix